src/mobile/src/main/java/tupperdate/android/ui/home/chats/OneConversation.kt

```diff
@@ -0,0 +1,119 @@
+package tupperdate.android.ui.home.chats
+
+import androidx.compose.material.Text
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumnFor
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import tupperdate.android.ui.theme.GreyBackgroundMessage
+import tupperdate.android.ui.theme.Smurf200
+import tupperdate.android.ui.theme.TupperdateTheme
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+data class Message(
+    val textMessage: String,
+    val iSend: Boolean
+)
+
+@Composable
+fun OneConversation(
+    msgList:List<Message>,
+    onReturnClick: () -> Unit,
+    imageOther: Any,
+    nameOther: String,
+    otherOnlineStatus: String,
+    onSendClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Scaffold(modifier = modifier,
+        topBar = {
+            ChatTopBar(
+                onReturnClick = onReturnClick,
+                imageOther = imageOther,
+                nameOther = nameOther,
+                otherOnline = otherOnlineStatus
+            )
+        },
+        bodyContent = { MessagesColumn(msgList = msgList) },
+        bottomBar = { ChatBottomBar(onSendClick = onSendClick) }
+    )
+}
+
+@Composable
+private fun MessagesColumn(
+    msgList:List<Message>,
+    modifier: Modifier = Modifier
+) {
+    LazyColumnFor(items = msgList,
+    modifier = modifier.padding(horizontal = 16.dp)) {
+        DisplayMessage(
+            message = it
+        )
+    }
+}
```

Comment:
```suggestion
private fun MessagesColumn(
    msgList:List<Message>,
    modifier: Modifier = Modifier
) {
    LazyColumn(modifier = modifier.padding(horizontal = 16.dp)) {
        items(msgList){
            DisplayMessage(it)
        }
    }
}
```

This will update the code to the new version of compose